galgebra/lt.py

```diff
@@ -9,9 +9,10 @@
 from copy import copy
 from functools import reduce
 
-from sympy import (
-    expand, symbols, Matrix, Transpose, zeros, Symbol, Function, S, Add
+from sympy.core.backend import (
+    expand, symbols, zeros, Symbol, Function, S, Add
 )
+from sympy import Matrix, Transpose
```

Comment:
If you use sympy's `Matrix`, expressions will be converted to SymPy and you'll lose the performance benefits of SymEngine

Reply:
I guessed so. The problem here is if I don't use sympy's Matrix, it will fail:

```
TypeError: can't set attributes of built-in/extension type 'symengine.lib.symengine_wrapper.MutableDenseMatrix'
```

due to lines like https://github.com/pygae/galgebra/blob/master/galgebra/printer.py#L628

Reply:
In the long run, we probably should not be attempting to override `__str__`

Reply:
I think this will be easier to fix once we drop support for Python 2.7 and rewrite the printer somehow.